src/applications/disability-benefits/rating-calculator/components/RatingRow.jsx

```diff
@@ -31,7 +31,7 @@ const RatingRow = ({
       <span id={rowId} className="sr-only">
         row {indx + 1}
       </span>
-      <div className="vads-l-col--2 vads-u-padding-right--2">
+      <div className="vads-l-col--3 vads-u-padding-right--2">
```

Comment:
The grid wasn't adding up correctly to 12, and was resulting in a label to wrap to the next line. This fixes that.